src/theory/strings/regexp_operation.cpp

```diff
@@ -994,7 +996,6 @@ Node RegExpOpr::reduceRegExpNegConcatFixed(Node mem, Node reLen, size_t index)
     }
   }
   Node r2 = nvec.size() == 1 ? nvec[0] : nm->mkNode(REGEXP_CONCAT, nvec);
-  r2 = Rewriter::rewrite(r2);
```

Comment:
Why is this rewrite removed?

Reply:
It wasn't necessary and actually will prevent us from doing proofs.